<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgIceCreamShop</class>
 <widget class="QDialog" name="dlgIceCreamShop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mali Medium</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bobo's Ice Cream Shop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mali Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose your Ice Cream</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mali Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose your beverage</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>491</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mali Medium</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpIceCream">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Ice Cream</string>
   </property>
   <widget class="QCheckBox" name="chkCookieDough">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cookie Dough $2.00</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkMintChip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mint Chip $2.25</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRockyRoad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rocky Road $2.75</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkSpumoni">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Spumoni $2.50</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Beverage</string>
   </property>
   <widget class="QCheckBox" name="chkScotch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scotch $3.00</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkSoda">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Soda $1.25</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkCoffee">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Coffee $1.50</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
